include/odp/api/spec/ipsec.h

```diff
@@ -1145,16 +1146,25 @@ typedef struct odp_ipsec_in_param_t {
 } odp_ipsec_in_param_t;
 
 /**
- * Outbound inline IPSEC operation parameters
+ * Outbound inline IPsec operation parameters
```

Comment:
You are right that IPsec is better spelling. However, I'd prefer that spelling would be changed once (for the entire file) in a separate patch.

Reply:
RFC (https://tools.ietf.org/html/rfc4301) deprecates the usage of IPSEC. So using IPSEC to in the new code added didn't seem right. I can switch to IPSEC for all usages. Or I can submit a parent patch fixing the entire file. Can you suggest what would be the right approach?

Reply:
I've replaced IPsec with IPSEC as part of v4.